src/ring_theory/dedekind_domain.lean

```diff
@@ -0,0 +1,63 @@
+/-
+Copyright (c) 2020 Kenji Nakagawa. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Kenji Nakagawa, Anne Baanen, Filippo A. E. Nuccio
+-/
+import ring_theory.fractional_ideal
+import ring_theory.ideal.over
+import ring_theory.discrete_valuation_ring
+import order.zorn
+
```

Comment:
... and [a module doc string](https://leanprover-community.github.io/contribute/doc.html#header-comment).

Reply:
This one's still missing.